bundles/org.openhab.binding.sonnen/README.md

```diff
@@ -0,0 +1,58 @@
+# Sonnen Binding
+
+The binding for sonnen communicates with a sonnen battery. More information about the sonnen battery can be found [here](https://sonnen.de/)
+
+## Supported Things
+
+| Things | Description  | Thing Type |
+|--------|--------------|------------|
+| sonnen | Control of a sonnen battery | battery|
+
+
+## Thing Configuration
+
+Only one parameter is required. The IP-Address of the sonnen battery in your local network.
+
+```
+Thing sonnen:battery:myBattery "sonnenBattery"  [ hostIP="192.168.0.10"]
```

Comment:
To be adjusted, depending on how you finally named your thing type.
With your current naming (see my other comment), the good syntax is:
`Thing sonnen:sonnenbattery:myBattery "Sonnen Battery"  [ hostIP="192.168.0.10"]`
And you can propose as label the final default label of your thing type.

Reply:
ok. adapted